.travis.yml

```diff
@@ -1,5 +1,4 @@
-# opt-in to Travis new infrastructure
-sudo: false
+sudo: true
```

Comment:
I need to be able to use `sudo` and similar commands. The main thing is to be able to `chmod` coursier. This is disallowed on the container version of Travis. The `sudo: true` puts it all on a virtual machine. At least this is my understanding.